src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -335,8 +335,23 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 
 <!-- Do not resolve from the GAC under any circumstances in Mobile -->
 <PropertyGroup>
-	<AssemblySearchPaths>$([System.String]::Copy('$(AssemblySearchPaths)').Replace('{GAC}',''))</AssemblySearchPaths>
-	<AssemblySearchPaths Condition="'$(MSBuildRuntimeVersion)' != ''">$(AssemblySearchPaths.Split(';'))</AssemblySearchPaths>
+	<AssemblySearchPaths>
+		{HintPathFromItem};
+		{TargetFrameworkDirectory};
+		{RawFileName};
+	</AssemblySearchPaths>
+	<AllowedReferenceAssemblyFileExtensions>
+		.dll;
+		.exe;
+	</AllowedReferenceAssemblyFileExtensions>
+	<AllowedReferenceRelatedFileExtensions>
+		.pdb;
+		.xml;
+		.dll.config;
+		.exe.config;
+		.dll.mdb;
```

Comment:
Do we *really* need to continue supporting `.mdb`, or should we just require Portable PDBs now?

Reply:
`.mdb` files are required when "classic PDB"/full-PDB files are used; see also commit dc8e1529b471e50b0cdf0137384a4667ad20419c.

Can we drop support for full-PDB files?

Reply:
If/when we drop support for full-PDB files, we can *also* [remove all use of `pdb2mdb`](https://github.com/xamarin/xamarin-android/blob/3f360c09739a2c83056bb6753d357e8b9b734de4/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj#L253-L330).

Reply:
I will remove `.exe.config` and `.exe.mdb` for now.

If we remove `mdb` or `full` `pdb`, we can trim this further later.